IDR/notebook/2.collect_statistics.ipynb

```diff
@@ -0,0 +1,416 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 15,
```

Comment:
Non-sequential execution count. (remember to nbconvert again once you adjust)

Reply:
@gwaybio It was my understanding that the execution count continues throughout the entire notebook directory. For example, the first `.ipynb` (`0.get_ids.ipynb`) file executes 1-7, the second (`1.describe_screens.ipynb`) executes 8-14, and then this one (`2.collect_statistics.ipynb`) executes 15-21. I do see that 19-21 are incorrect, but here you're referencing number 15. I'll fix this, but just checking that execution counts carry over from notebook to notebook!

Reply:
They do not carry over. Numbering restarts at each notebook

Reply:
@gwaybio sounds good. I'll fix that for the other notebooks as well.

Reply:
@gwaybio Execution orders were fixed.

Added in b621aff61a32808ce4088a415464a8b4ae3b4aea for `1.describe_screens.ipynb` and ba8bc8f4ed4ddf990a6cd260d9985ddfe4e9b2a2 for `2.collect_statistics.ipynb`.